portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase

```diff
@@ -153,4 +153,134 @@ definition {
 			value1 = '''<div class="fragment_1">Fragment Name</div>''');
 	}
 
+	@description = "This is a test case for LPS-129852. Enabled Propagate Fragment changes should not affect Collection Fragment in Staging and Live Site."
```

Comment:
View the changes of fragment on content page still persist in staging and live site after overwrite the fragment